balancer-js/src/modules/pools/pool-types/concerns/composableStable/exit.concern.integration.spec.ts

```diff
@@ -0,0 +1,182 @@
+const signer = provider.getSigner();
+
+// Slots used to set the account balance for each token through hardhat_setStorageAt
+// Info fetched using npm package slot20
+const initialBalance = '100000';
+const slippage = '0'; // 1%
+
+const pool = pools_16350000.find(
+  (pool) =>
+    pool.id ==
+    '0xa13a9247ea42d743238089903570127dda72fe4400000000000000000000035d' // Balancer Aave Boosted StablePool
+) as unknown as Pool;
+const tokensOut = pool.tokens.filter(({ address }) => address !== pool.address);
+const controller = Pools.wrap(pool, networkConfig);
+
+describe('exit stable pools execution', async () => {
+  let amountsOut: string[];
+  let transactionReceipt: TransactionReceipt;
+  let bptBalanceBefore: BigNumber;
+  let bptBalanceAfter: BigNumber;
+  let bptMaxBalanceDecrease: BigNumber;
+  let tokensBalanceBefore: BigNumber[];
+  let tokensBalanceAfter: BigNumber[];
+  let tokensMinBalanceIncrease: BigNumber[];
+  let signerAddress: string;
+
+  // Setup chain
+  before(async function () {
+    await forkSetup(
+      signer,
+      pool.tokensList,
+      Array(pool.tokensList.length).fill(0),
+      Array(pool.tokensList.length).fill(
+        parseFixed(initialBalance, 18).toString()
+      ),
+      jsonRpcUrl as string,
+      16350000 // holds the same state as the static repository
+    );
+    signerAddress = await signer.getAddress();
+  });
+
+  const testFlow = async (
+    [to, data, maxBPTIn, minAmountsOut]: [string, string, string, string[]],
+    exitTokens: string[]
+  ) => {
+    // Check balances before transaction to confirm success
+    [bptBalanceBefore, ...tokensBalanceBefore] = await getBalances(
+      [pool.address, ...exitTokens],
+      signer,
+      signerAddress
+    );
+    // Get expected balances out of transaction
+    bptMaxBalanceDecrease = BigNumber.from(maxBPTIn);
+    tokensMinBalanceIncrease = minAmountsOut.map((a) => BigNumber.from(a));
+    // Send transaction to local fork
+    const transactionResponse = await signer.sendTransaction({
+      to,
+      data,
+      gasLimit: 3000000,
+    });
+    transactionReceipt = await transactionResponse.wait();
+
+    // Check balances after transaction to confirm success
+    [bptBalanceAfter, ...tokensBalanceAfter] = await getBalances(
+      [pool.address, ...exitTokens],
+      signer,
+      signerAddress
+    );
+  };
+
+  context('exitExactBPTIn', async () => {
+    context('single token max out', async () => {
+      before(async function () {
+        const bptIn = parseFixed('10', 18).toString();
+        const { to, data, minAmountsOut } = controller.buildExitExactBPTIn(
+          signerAddress,
+          bptIn,
+          slippage,
+          false,
+          pool.tokensList.filter((address) => address !== pool.address)[0]
+        );
+        await testFlow(
+          [to, data, bptIn, minAmountsOut],
+          pool.tokensList.filter((address) => address !== pool.address)
```

Comment:
Can you use `tokensOut.map((t) => t.address)` like you did below. I think its clearer.